django/embedded_analytics/README.md

```diff
@@ -0,0 +1,33 @@
+# Django app
+
+This Django application demonstrates a simple, barebones Metabase dashboard embedded in a web application.
+
+## Prerequisites
+
+- **Metabase**. You should have already completed the setup detailed in the [README](../README.md) for this repository, which shows you how to get an instance of Metabase up and running in the [metabase](../metabase) directory of this repository.
+
+- **Django**. You'll need to have [Django installed](https://docs.djangoproject.com/en/3.0/topics/install/) on your machine.
```

Comment:
The `pip install` command below will install Django so we probably just need to tell folks they need Python installed.